function/services/application-mapper/index.test.js

```diff
@@ -40,6 +40,50 @@ describe('Application mapper', () => {
         expect(applicationFormJson.split_funeral).toBeTruthy();
     });
 
+    describe('Set the channel column with the application source', () => {
+        it('Should default channel column to "W" if not value is provided', async () =>{
+            applicationSummaryJson = fs.readFileSync(
+                'function/resources/testing/application-meta.json'
+            );
+            oracleObject = await mapApplicationDataToOracleObject(
+                JSON.parse(applicationSummaryJson),
```

Comment:
Just `require` it instead of using `fs`. saves on having to actively parse things.